doc/Changelog.md

```diff
@@ -1,13 +1,15 @@
-# Changelog
+# Changelog {#Changelog}
 
-# git master {#master}
+# git master
+
+# Release 1.3.0 (2-Nov-2015)
```

Comment:
Others where month... not sure which one to converge to. Name is more internationally friendly.